install_scripts/install_lago.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -xe
 
+#Let's reviews the install script
 
 REPONAME="ovirt-system-tests"
 REPOURL="https://gerrit.ovirt.org"
```

Comment:
I'll add the comments here, as github does not allow commenting on the 'hidden' code :)

* the usage of 'function' for bash function is not recommended, as it's not as compatible as the other syntax (I think that old bash versions and other shells, not sure if it's an issue though)
* Use `grep -q` instead of just `grep`, as that will not show any output.
* +1 for the usage of local :wine_glass: 
* To configure the repos, you wight want to check the repomanage utilities, though it might not be supported on old el
* Missing a couple of quoted variable (specially $HOME and other non-locally generated ones)
* Hmmm, adding a user to a group will not be enough without relogging in again, you might want to run sudo or similar after (newgrp might be useful there too...)
* Also you will have to rmmod + insmod the intel_kvm after enabling nested, or reboot (rebooting will force the user to update the groups permissions too), maybe if it has to configure it, just showing a message with 'reboot needed please' will be enough.
* If the user is not root, you might need to sudo to chnge groups and enable/install stuff.

Great initiative!